Wox.Plugin.Runner/Settings/RunnerSettingsViewModel.cs

```diff
@@ -21,9 +21,9 @@ public void LoadCommands()
                 RunnerConfiguration.Commands.Select( c => new CommandViewModel( c ) ) );
         }
 
-        public ObservableCollection<CommandViewModel> Commands { get; set; }
+        public ObservableCollection<CommandViewModel>? Commands { get; set; }

```

Comment:
Maybe refactor a bit to make this not nullable. The viewmodel is not intended to have null Commands.